interceptors/DataApiInterceptors.cfc

```diff
@@ -95,15 +95,16 @@ component extends="coldbox.system.Interceptor" {
 		if ( !IsSimpleValue( restRequest ) ) {
 			var api      = restRequest.getApi();
 			var resource = restRequest.getResource();
+			var regex    = "^data\.v1\.(WholeEntity|SingleRecord|Queue)";
 
-			if ( api == "/data/v1" && resource.count() ) {
+			if ( api == "/data/v1" && reFindNoCase( regex, resource.handler ?: "" ) && resource.count() ) {
 				dataApiService.onRestRequest( restRequest, restResponse );
 				return;
 			}
 
 			var dataApiRoutes = dataApiConfigurationService.getDataApiRoutes();
 			for( var apiRoute in dataApiRoutes ) {
-				if ( api == apiRoute && reFindNoCase( "^data\.v1", resource.handler ?: "" ) ) {
+				if ( api == apiRoute && reFindNoCase( regex, resource.handler ?: "" ) ) {
```

Comment:
The check on namespaced APIs is not necessarily required, because the resource handler differs anyway. I have just added it here to be consistent with the default API.